### DISCUSSION TITLE
Include Meteo France to Fdroid version ?

### DISCUSSION THREAD

**Discussion by CelianGdfrd** [breezy-weather/breezy-weather#909] [Q&A]
### Describe your idea

Meteo France recently shifted all its APIs and data to libre license (Etalab, similar to CC-BY 4.0).

Would this be sufficient to put it in the Fdroid version ?

I can help on legal questions you may have.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] My idea is not related to a question covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document, or a feature described in the [Homepage](https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md) document.
- [X] I have updated the app to **[version 5.1.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
The F-Droid version is a flavor of Breezy Weather with no non-free and non-self hostable sources. If Météo-France releases its source code, I can include it in this flavor. Otherwise, just use the Standard flavor?

**Reply by papjul** [reply]
Additionally, the fdroid flavor is used by the F-Droid default repo which forbids non-public API keys
